src/js/views/createDevice/steps/SummaryStep/SummaryStep.jsx

```diff
@@ -86,34 +100,54 @@ const SummaryStep = ({
               </Typography>
             </Box>
           </Box>
-
-          <Box className={classes.certificateData}>
-            <Typography>{t('summaryStep.certificate')}</Typography>
-            <Link href='https://www.google.com' download>
-              Link
-            </Link>
-          </Box>
-
-          <Box className={classes.certificateData}>
-            <Typography>{t('summaryStep.privateKey')}</Typography>
-            <Link href='https://www.google.com' download>
-              Link
-            </Link>
-          </Box>
-
-          <Box className={classes.certificateData}>
-            <Typography>{t('summaryStep.publicKey')}</Typography>
-            <Link href='https://www.google.com' download>
-              Link
-            </Link>
-          </Box>
-
-          <Box className={classes.certificateData}>
-            <Typography>{t('summaryStep.caCertificate')}</Typography>
-            <Link href='https://www.google.com' download>
-              Link
-            </Link>
-          </Box>
+          {selectedCertificate?.pem ? (
+            <Box className={classes.certificateData}>
+              <Typography>{t('summaryStep.certificate')}</Typography>
+              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
Maybe you can disable this rule in the entire file or find a way to style the element as a Link without using a LInk, so the Eslint will not give you an error. Read more about this rule here: [anchor-is-valid.md](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md)